src/oc_erchef/apps/oc_chef_wm/src/chef_wm_prometheus_json_format.erl

```diff
@@ -0,0 +1,100 @@
+-module(chef_wm_prometheus_json_format).
+
+-export([
+         content_type/0,
+         format/0,
+         format/1]).
+
+-include_lib("prometheus/include/prometheus_model.hrl").
+
+-define(PROCESS_DICT_STORAGE, chef_wm_prometheus_json_format_data).
+
+-spec content_type() -> binary().
+content_type() ->
+    <<"application/json">>.
+
+-spec format() -> binary().
+format() ->
+    format(default).
+
+-spec format(Registry :: prometheus_registry:registry()) -> binary().
+format(Registry) ->
+    put(?PROCESS_DICT_STORAGE, []),
```

Comment:
My only concern with using the process dictionary here is out of ignorance.  I don't know if putting what could be a lot of data in the process dictionary will cause us any problems down the road.  The prometheus code uses the ram_disk module for all of it's formatters; however I have no idea if that is better or worse.  Perhaps for now this is just something we live with but keep an eye on.

Reply:
Yea, I wanted to use ram file, but I couldn't figure out how to do it without rolling my own json serializer. That being said, there shouldn't be a lot of data. The other option I could think of was creating a gen server to store the state.

If it makes you feel better, webmachine makes fairly heavy use of the process dictionary: https://github.com/webmachine/webmachine/blob/82472528d8a735d28fa92a7c6e03fc97cb4d1913/src/webmachine_decision_core.erl#L28

Reply:
I'm +1 on leaving this as is for now.  Thanks for following up.